src/v2/Components/Artwork/FillwidthItem.tsx

```diff
@@ -135,7 +130,8 @@ export class FillwidthItemContainer extends React.Component<
             }}
           >
             <Image
-              src={this.getImageUrl()}
+              src={this.getImageUrl(1)}
+              srcSet={`${this.getImageUrl(1)} 1x, ${this.getImageUrl(2)} 2x`}
```

Comment:
This fixes images such that the browser will just load the version depending on ur pixel ratio, vs trying to download both.